src/resources/views/components/mail/grid.blade.php

```diff
@@ -8,21 +8,21 @@
                 <td align="center" valign="top">
                     <table {{ $attributes->merge(['class' => "container grid grid-col-$columns"]) }} width="600" cellpadding="0" cellspacing="0" border="0">
                         <tr>
-                            <td class="mobile" align="center" valign="top">
+                            <td class="mobile" align="center" valign="top" {{ $one?->attributes }}>
```

Comment:
This wont work as the component will simply use the default class, which in this case is 'mobile'. In order for this to work, you would need to merge the default class with whatever is passed when calling a component:

```blade
<td {{ $one?->attributes->merge(['class' => 'mobile']) }} align="center" valign="top" >
    {{ $one ?: $slot }}
</td>
```

Reply:
It does work if you want to inject something else than class, like for example raw style. But I get your point with class, yeah.